### ISSUE TITLE
404s suggesting wrong template names

### ISSUE THREAD

**Issue by trvsdnn** [trvsdnn/frank#2]
Seems to be mainly be happening with index and a few other edge cases.

**Comment by jacquescrocker** [comment]
You mean the msg "Try creating index.html.haml"?

Its supposed to be just `index.haml` right?

Can we make it support both? index.html.haml and index.haml

**Comment by trvsdnn** [comment]
Yeah, that's the one. I forgot about this bug, it's pretty old. I sort of thought I might end up taking away the suggestions on the 404s. Do you have an opinion on the issue?

Either way, i'll figure something out for the next release.

**Comment by jacquescrocker** [comment]
I think it should support both .html.haml and .haml. I usually name my rails templates .html.haml so it'd make it a bit easier to swap between the 2

**Comment by trvsdnn** [comment]
That's another issue. It might break some stuff adding support for .html.haml/erb extensions. I would like to get that added though.

Do you think having the 404 suggest templates is important though?

**Comment by jacquescrocker** [comment]
oh, umm.. nah. i dont think its important to suggest templates
